server/api/login.go

```diff
@@ -183,29 +183,34 @@ func HandleAuth(c *gin.Context) {
 	// create or set the user's organization if it isn't linked yet
 	if user.OrgID == 0 {
 		// check if an org with the same name exists already and assign it to the user if it does
-		if org, err := _store.OrgFindByName(user.Login); err == nil && org != nil {
+		// TODO: find the org by name and forgeID directly
+		org, err := _store.OrgFindByName(user.Login)
```

Comment:
I've noticed this bug as the error was not handled by the `if` below as it was inline in the previous if.

Reply:
Good catch